plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/core/resources/MavenProject.java

```diff
@@ -28,4 +30,12 @@ public MavenProject(TreeItem item) {
 	public String[] getNatureIds() {
 		return new String[] { "org.eclipse.m2e.core.maven2Nature" };
 	}
+	
+	public void updateMavenProject() {
```

Comment:
can you please add updateMavenProject(TimePeriod waitForJobs) which would call updateProject.ok(TimePeriod). See my comment below